api/src/operations/mail/index.ts

```diff
@@ -14,9 +14,11 @@ export default defineOperationApi<Options>({
 
 	handler: async ({ body, to, type, subject }, { accountability, database, getSchema }) => {
 		const mailService = new MailService({ schema: await getSchema({ database }), accountability, knex: database });
-
+		// If 'body' is of type object/undefined (happens when body consists solely of a placeholder)
+		// convert it to JSON string
+		const safeBody = typeof body === 'object' || typeof body === 'undefined' ? JSON.stringify(body) : body;
```

Comment:
While `object`/`null` covers most bases here the mailservice will still fail when being passed a `Number` directly (possibly for a `Function` too)a.
```suggestion
		const safeBody = typeof body !== 'string' ? JSON.stringify(body) : body;
```

Reply:
Thanks @br41nslug! I like that suggestion 👍🏻